src/net/sourceforge/kolmafia/request/AdventureRequest.java

```diff
@@ -390,7 +405,8 @@ public static final String registerEncounter(final GenericRequest request) {
           };
     }
 
-    String prettyEncounter = StringUtilities.getEntityDecode(encounter);
+    String prettyEncounter = stripDevReadout(StringUtilities.getEntityDecode(encounter));
```

Comment:
Conceptually I would rather only do this if mafia knows it is running on the dev server.  But there are very few (two?) places where mafia knows that and, in the absence of a profiler, I can certainly understand that checking might be as expensive as just running stripDevReadout unconditionally.  A comment, not a blocker.

Reply:
I can see if it's easy to check for that!